pkgs/development/python-modules/google-auth-oauthlib/default.nix

```diff
@@ -28,6 +29,8 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  pytestFlagsArray = lib.optionals stdenv.isDarwin ["-k 'not test_run_local_server'"];
```

Comment:
```suggestion
  disabledTests = lib.optionals stdenv.isDarwin [ "not test_run_local_server" ];
```

Reply:
I added this to the google-auth-oauthlib commit.